src/pages/workspace/distanceRates/PolicyDistanceRatesPage.tsx

```diff
@@ -0,0 +1,280 @@
+            });
+        }
+
+        return options;
+    };
+
+    const headerButtons = (
+        <View style={[styles.w100, styles.flexRow, isSmallScreenWidth && styles.mb3]}>
+            {selectedDistanceRates.length === 0 ? (
+                <>
+                    <Button
+                        medium
+                        text={translate('workspace.distanceRates.addRate')}
+                        onPress={addRate}
+                        style={[styles.mr3, isSmallScreenWidth && styles.flexGrow1]}
+                        icon={Expensicons.Plus}
+                        iconStyles={[styles.mr2]}
+                        success
+                    />
+
+                    <Button
+                        medium
+                        text={translate('workspace.common.settings')}
+                        onPress={openSettings}
+                        style={[isSmallScreenWidth && styles.flexGrow1]}
+                        icon={Expensicons.Gear}
+                        iconStyles={[styles.mr2]}
+                    />
+                </>
+            ) : (
+                <ButtonWithDropdownMenu<WorkspaceDistanceRatesBulkActionType>
+                    shouldAlwaysShowDropdownMenu
+                    pressOnEnter
+                    customText={`${selectedDistanceRates.length} ${translate('workspace.distanceRates.selected')}`}
+                    buttonSize={CONST.DROPDOWN_BUTTON_SIZE.MEDIUM}
+                    onPress={() => null}
+                    options={getBulkActionsButtonOptions()}
+                    buttonRef={dropdownButtonRef}
+                    style={[isSmallScreenWidth && styles.flexGrow1]}
+                    wrapperStyle={styles.w100}
+                />
+            )}
+        </View>
+    );
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={route.params.policyID}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={route.params.policyID}>
+                <ScreenWrapper
+                    includeSafeAreaPaddingBottom={false}
+                    style={[styles.defaultModalContainer]}
+                    testID={PolicyDistanceRatesPage.displayName}
+                    shouldShowOfflineIndicatorInWideScreen
+                >
+                    <HeaderWithBackButton
+                        icon={Illustrations.CarIce}
+                        title={translate('workspace.common.distanceRates')}
+                        shouldShowBackButton={false}
+                    >
+                        {!isSmallScreenWidth && headerButtons}
+                    </HeaderWithBackButton>
+                    {isSmallScreenWidth && <View style={[styles.ph5]}>{headerButtons}</View>}
+                    <Text style={[styles.pl5, styles.pb2, styles.pt4, styles.textSupporting]}>{translate('workspace.distanceRates.centrallyManage')}</Text>
+                    <SelectionList
+                        canSelectMultiple
+                        ListItem={RadioListItem}
```

Comment:
```suggestion
                        ListItem={TableListItem}
```

@MrMuzyk the implementation differs from the design, is this intentional?

![Screenshot 2024-03-06 at 10 03 57 at night](https://github.com/Expensify/App/assets/75031127/0bee4a1d-14c4-4cf6-87cf-046db7c72da4)

Reply:
@getusha Changing list item like you've suggested should do the trick. Unless you've meant that something else looks off?

Reply:
thats it, thanks!